my-app/src/components/moves.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const moves = (history, jumpTo) =>
```

Comment:
I couldn't see where `history` and `jumpTo` come from in the implementation, currently `moves` function is called without any arguments